slideshow/src/main/java/nl/ou/domain/StopStrategy.java

```diff
@@ -1,7 +1,9 @@
 package nl.ou.domain;
 
+import nl.ou.services.GUIFacade;
```

Comment:
Nu refereert het domain aan een class in de services laag, dan moet of dit naar de services of GUIFacade interface het domain in.

Reply:
GUIFacade naar het domein lijkt me niet goed. StopStrategy in het services package lijkt me helemaal prima. Wordt aangepast.